### ISSUE TITLE
Error when executing npm run dev

### ISSUE THREAD

**Issue by Isatol** [vuetifyjs/webpack#11]
Everything is fine at start, but when i stopping the service, an run it again. This error appear:

**> gaxtank@1.0.0 dev C:\Proyecto\GaxTank
> webpack-dev-server --inline --progress --config build/webpack.dev.conf.js

 94% asset optimization

 ERROR  Failed to compile with 2 errors                                                                                                                                       12:31:57
This dependency was not found:

* vuetify/dist/vuetify.min.css in ./src/main.js

To install it, you can run: npm install --save vuetify/dist/vuetify.min.css

This relative module was not found:

* ./assets/css/animate.css in ./src/main.js**

How to solve it?

**Comment by nekosaur** [comment]
It looks like you've added an import for animate.css? Does it actually exist at the path you've specified?

**Comment by Isatol** [comment]
@nekosaur Yes the animate.css is in the assets/css/ folder, this case only happens to me when i finish the server and start it again
